jp/components/drop_down.md

```diff
@@ -0,0 +1,283 @@
+
+```html
+<!-- dropdown.component.html -->
+
+<div class="sample-wrapper drop-down-wrapper">
+    <button igxButton="raised" (click)="toggleDropDown()">Options</button>
+    <igx-drop-down>
+        <igx-drop-down-item *ngFor="let item of items">
+            {{ item.field }}
+        </igx-drop-down-item>
+    </igx-drop-down>
+</div>
+```
+
+```typescript
+// dropdown.component.ts
+
+...
+    @ViewChild(IgxDropDownComponent) public igxDropDown: IgxDropDownComponent;
+
+    public items: any[] = [];
+
+    constructor() {
+        for (let i = 1; i < 4; i ++) {
+            const item = { field: "Option " + i };
+            this.items.push(item);
+        }
+    }
+
+    public ngOnInit() {
+    }
+
+    public toggleDropDown() {
+        this.igxDropDown.toggle();
+    }
+
+```
+
+
+サンプルを正しく構成すると、複数のオプションを含むドロップダウンが表示されます。
+
+<div class="sample-container" style="height: 200px">
+    <iframe id="dropdown-sample-1-iframe" src='{environment:demosBaseUrl}/dropdown-sample-1' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn"   data-iframe-id="dropdown-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">                view on stackblitz
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+選択済みの項目をあらかじめ定義する場合、[**IgxDropDownComponent**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop_down.html) **igx-drop-down**  `onOpening` イベントを処理します。
+
+```html
+<!-- dropdown.component.html -->
+
+<div class="sample-wrapper drop-down-wrapper">
+    <button igxButton="raised" (click)="toggleDropDown()">Options</button>
+    <igx-drop-down (onOpening)="onOpening($event)">
+        <igx-drop-down-item *ngFor="let item of items">
+            {{ item.field }}
+        </igx-drop-down-item>
+    </igx-drop-down>
+</div>
+```
+
+```typescript
+// dropdown.component.ts
+
+...
+    @ViewChild(IgxDropDownComponent) public igxDropDown: IgxDropDownComponent;
+
+    public items: any[] = [];
+
+    constructor() {
+        for (let i = 1; i < 4; i ++) {
+            const item = { field: "Option " + i };
+            this.items.push(item);
+        }
+    }
+
+    public ngOnInit() {
+    }
+
+    public toggleDropDown() {
+        this.igxDropDown.toggle();
+    }
+
+    public onOpening(ev) {
+        this.igxDropDown.setSelectedItem(0);
+    }
+
+```
+
+<div class="sample-container" style="height: 200px">
+    <iframe id="dropdown-sample-2-iframe" src='{environment:demosBaseUrl}/dropdown-sample-2' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn"   data-iframe-id="dropdown-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">                stackblitz で開く
```

Comment:
Same comment as before for the button text casing.

Reply:
Replaced stackblitz with StackBlitz.